mobile/src/main/java/ch/epfl/sdp/mobile/ui/prepare_game/OpponentList.kt

```diff
@@ -0,0 +1,117 @@
+package ch.epfl.sdp.mobile.ui.prepare_game
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyListState
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.compositeOver
+import androidx.compose.ui.unit.dp
+import ch.epfl.sdp.mobile.state.toColor
+import ch.epfl.sdp.mobile.ui.social.Person
+
+/**
+ * Composable that represents a list of potential opponents, with one potentially selected
+ * @param state the state of the [PrepareGameScreen]
+ * @param modifier [Modifier] for this composable
+ * @param lazyListState the state of the lazyList displaying opponents
+ * @param key a function which uniquely identifies the list items.
+ * @param P the type of the [Person].
```

Comment:
Still the old docs / parameters I think

Reply:
[7604386](https://github.com/epfl-SDP/android/pull/200/commits/7604386e86b16c6001aaa3e7b03a38b37ac08991)